app/mailers/oral_history_delivery_mailer.rb

```diff
@@ -0,0 +1,70 @@
+class OralHistoryDeliveryMailer < ApplicationMailer
+
+  def oral_history_delivery_email
+    raise ArgumentError.new("Required params[:request] missing") unless request.present?
+    raise ArgumentError.new("Required patron email missing") unless request.patron_email.present?
+    mail(to: to_address, subject: subject, content_type: "text/html")
+  end
+
+  def request
+    @request ||= params[:request]
+  end
+
+  def to_address
+    request.patron_email
+  end
+
+  # Note: any value greater than 604800 will raise an exception.
+  # See https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/S3/Presigner.html
+  def how_long_urls_will_be_valid
+    1.week.to_i - 10
+  end
+
+
+  def created_at
+    request.created_at
+  end
+
+  def patron_name
+    request.patron_name
+  end
+
+  def work
+    request.work
+  end
+
+  def assets
+    work.members.order(:position).select {|x| x.is_a? Asset}
```

Comment:
we shouldn't be sending all assets, but only those marked for restricted delivery by request. All assets might include for instance the public "front matter" PDF.  i forget where in the data the mark is for "available by request", but it's somewhere or other. 

There should be a test that verifies this too. 

Nice job catching `order(:position)`, I actually came here to confirm that.

Reply:
Fixed that. Makes sense.